tesseract-core/src/schema.rs

```diff
@@ -274,6 +288,32 @@ impl Cube {
         }
         None
     }
+
+    pub fn get_child_level(&self, level_name: &LevelName) -> Result<Option<Level>, Error> {
+        let hierarchy =  match self.get_hierarchy(level_name) {
+            Some(h) => h,
+            None => {
+                return Err(format_err!("Could not find parent hierarchy for level: {}", level_name.level))
```

Comment:
This is a little shorter using `self.get_hierarchy().ok_or_else(|| format_err!())?`